src/Main.elm

```diff
@@ -1501,7 +1520,7 @@ decodeInputField =
                     "ShortText" ->
                         Json.Decode.succeed ShortText
                             |> andMap (Json.Decode.field "inputType" Json.Decode.string)
-                            |> andMap (Json.Decode.field "maxLength" (Json.Decode.nullable Json.Decode.int))
+                            |> andMap (Json.Decode.field "attributes" (Json.Decode.keyValuePairs Json.Decode.string))
```

Comment:
to avoid `"attributes"` key nesting, we can

```suggestion
                            |> andMap (Json.Decode.keyValuePairs Json.Decode.string |> Json.Decode.map (excludeKeys ["type", "inputType"]))
```

where `excludeKeys` is `List comparable -> List (comparable, v) -> List (comparable, v)`

Reply:
actually nesting is necessary since we're already using "type" to encode InputField variants 🫣